### ISSUE TITLE
some questions，thanks

### ISSUE THREAD

**Issue by BelieferQAQ** [282857341/nnFormer#84]
1.Do you have to use python3.6? Is python3.7 OK？
2.How to apply it to your own data set (ribfrac). Each CT contains only 1-3 types of rib fracture (there are 5 types of rib fracture in the whole data set). Can nnformer be used？

**Comment by 282857341** [comment]
1. All experiments are done under the environment of python 3.6 , but I guess python 3.7 is ok.
2. Does this mean that a single case may not contain all segmentation classes？. If so, this does not affect the use of nnformer.

**Comment by BelieferQAQ** [comment]
> 

thank you very much

**Comment by BelieferQAQ** [comment]
How to train my own data set (ribfrac), a CT data set containing five kinds of rib fractures

**Comment by puppy2000** [comment]
> How to train my own data set (ribfrac), a CT data set containing five kinds of rib fractures

yes,i also want to know how. i want to train nnformer on may own dataset too
